CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ### 1.4.1 (Next)
-* Your contribution here.
+* [#97](https://github.com/dblock/iex-ruby-client/pull/97): Update READme.md for histroical daily request date parsing issue - [@umer_frooq](https://github.com/UmerFrooq).
```

Comment:
This should be 98 since you opened a new PR.

Reply:
sorry, it's first to contribute in the repo :)